api/build.gradle

```diff
@@ -249,6 +249,9 @@ dependencies {
     compile('io.reflectoring.diffparser:diffparser:1.4')
 
     compile 'dnsjava:dnsjava:3.4.3'
+
+    //Template engine
+    compile group: 'org.freemarker', name: 'freemarker', version: '2.3.23'
```

Comment:
Could you please try to use `Velocity` - we already have in the dependencies, so I'd prefer not to add one more template manager here.